setup.py

```diff
@@ -0,0 +1,36 @@
+from setuptools import find_packages
+from setuptools import setup
+from pip._internal.req import parse_requirements
+import os
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+
+# Pulls pip packages with versions from the requirements file
+install_requires = parse_requirements("requirements.txt", session="snowchange")
+test_requires = parse_requirements("requirements.txt", session="snowchnage")
+
+setup(
+    name="snowchange",
+    version="0.0.2",
```

Comment:
Can the version number match our current version number?

Reply:
Yes, it's now adjusted to 2.1.0